x-pack/platform/plugins/shared/cases/public/components/connectors/validate_json.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ValidationFunc } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+import { isEmpty, isObject } from 'lodash';
+import * as i18n from './translations';
+
+const MAX_ADDITIONAL_FIELDS_LENGTH = 10;
+
+export const validateJSON =
```

Comment:
nit: potentially rename to `getJSONValidator`

Reply:
👍 https://github.com/elastic/kibana/pull/236144/commits/f399c5385446667d2c35d2d4ee91f418a6242ba1